static/app/views/organizationIntegrations/sentryAppExternalForm.tsx

```diff
@@ -38,10 +37,10 @@ type State = Omit<Config, 'uri'> & {
 
 type Props = {
   api: Client;
-  sentryAppInstallation: SentryAppInstallation;
+  sentryAppInstallationUuid: string;
   appName: string;
   config: Config;
-  action: 'create' | 'link';
+  action: 'create' | 'update' | 'link';
```

Comment:
when is `update` used?

Reply:
The plan is for this to be used when updating an alert rule, though I'm not sure theres an PUT endpoint for it just yet.